soapysdr-sys/build.rs

```diff
@@ -144,14 +141,9 @@ fn main() {
         .size_t_is_usize(true)
         .header("wrapper.h");
 
-    let mut cc_builder = cc::Build::new();
-    cc_builder.file("wrapper.c");
```

Comment:
You're correct that this is no longer necessary if dropping support for 0.7, but needs further cleanup:

* Delete the wrapper.c file since it is no longer built.
* Remove the `_rust_wrapper_SoapySDRDevice_setupStream` from `wrapper.h`
* Un-blocklist `SoapySDRDevice_setupStream` below
* Remove the re-export in `lib.rs`
* Update the pre-generated generated bindings accordingly

Reply:
I'm happy to do if you want to drop support, but I believe the bindings offered here would still work for 0.7. I generally tend to err on the side of more compatibility, so I'm happy to do any extra steps to preserve 0.7 if you think any are needed (and/or worth it).